quarkus-picocli/src/test/java/ProdiIT.java

```diff
@@ -0,0 +1,29 @@
+import static java.util.concurrent.CompletableFuture.runAsync;
+
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.QuarkusApplication;
+
+@Disabled ("No possible to test 2 profiles with the same build")
```

Comment:
Why do we adding disabled test? Is there an existing issue?

Reply:
I did an investigation and is not possible to have both tests classes enable because one of them will fail depending on the profile activated.
Because Quarkus uses build-time profiles to include or exclude beans and configurations during the build process, so with annotations with `@IfBuildProfile("prod")` are only included in the build if the 'prod' or 'dev' profile is active at build time. 
If we remove the application.porperties with quarkus.profile config, you can test through mvn commands` -Dquarkus.profile=dev` or pro, and the tests will pass.

Reply:
The `quarkus.profile` is runtime property, our FW is not aware that you needed new custom build due to the runtime property. IMO either improve FW or workaround it by setting some build-time property that will differ between test classes.